config-overrides.js

```diff
@@ -3,7 +3,14 @@ const rewireReactHotLoader = require('react-app-rewire-hot-loader')
 const manualOverrides = require('./webpack.config')
 
 /* config-overrides.js */
-module.exports = function override(config, env) {
-  config = mixinDeep(config, manualOverrides)
-  return rewireReactHotLoader(config, env)
+module.exports = {
+  webpack: function override(config, env) {
+    config = mixinDeep(config, manualOverrides)
+    return rewireReactHotLoader(config, env)
+  },
+  jest: function(config) {
```

Comment:
Jest (the test runner) has a separate config; this lets it read out non-relative module paths

Reply:
I think this excellent explanation belongs in the code rather than this PR.